provider/resources.go

```diff
@@ -5865,7 +5868,11 @@ func setupComputedIDs(prov *tfbridge.ProviderInfo) {
 	prov.Resources["aws_backup_restore_testing_plan"].ComputeID = func(ctx context.Context, state resource.PropertyMap) (resource.ID, error) {
 		return attr(state, "name"), nil
 	}
-	prov.Resources["aws_backup_restore_testing_selection"].ComputeID = func(ctx context.Context, state resource.PropertyMap) (resource.ID, error) {
-		return attr(state, "restoreTestingPlanName", "name"), nil
+	prov.Resources["aws_backup_restore_testing_selection"].ComputeID = func(
+		ctx context.Context, state resource.PropertyMap,
+	) (resource.ID, error) {
+		// The ${name}:${restoreTestingPlanName} is the format used by TF import, so we are going to use it for
+		// the Pulumi ID as well.
+		return attrWithSeparator(state, ":", "name", "restoreTestingPlanName"), nil
```

Comment:
I wasn't even that worried about the separator, mostly the order. But it's fine nevertheless!

Reply:
Might as well, I think.